packages/flutter_tools/test/general.shard/widget_preview/preview_manifest_test.dart

```diff
@@ -17,6 +17,7 @@ import 'package:test/test.dart';
 import '../../src/common.dart';
 import '../../src/context.dart';
 
+// TODO(bkonyi): test pubspec change detection for workspaces
```

Comment:
These tests should be rewritten to utilize some of the utilities that have been added for testing widget preview projects. I'll do this in a follow up to avoid making this PR even larger.